.atomist/build/travis-build.bash

```diff
@@ -150,7 +150,11 @@ function main () {
             err "failed to create git tag: $git_tag"
             return 1
         fi
-        if ! git push --quiet --tags "https://$GITHUB_TOKEN@github.com/$TRAVIS_REPO_SLUG" > /dev/null 2>&1; then
+        local remote=origin
+        if [[ $GITHUB_TOKEN ]]; then
+            remote=https://$GITHUB_TOKEN@github.com/$TRAVIS_REPO_SLUG
```

Comment:
If this were a regular Bourne shell script, yes, but it is not required in BASH. BASH parses the whole line before expanding the variables, so even if the values of the variables have spaces in them, the assignment behaves properly.